vmdb/Gemfile

```diff
@@ -42,7 +42,13 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
 gem "default_value_for",              "~>2.0.3"
-gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+
+if RUBY_PLATFORM == 'java'
+  gem "puma"
+else
+  gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+end
```

Comment:
Ya, possibly.  Puma is the most popular threaded server, Unicorn is probably the most popular forking server.  We used Unicorn lots in production at previous companies.  I've never run Puma in production, [but it's what Heroku recommends](https://devcenter.heroku.com/changelog-items/594).

Puma has the advantage that it can do forking and threading models, and if we run on JRuby, we'll want to use the threaded model.